fennec/apps/integration/importer/importer.py

```diff
@@ -0,0 +1,108 @@
+from uuid import uuid4
```

Comment:
Technically it is just an importer, since it receives schema model as parameter. It can be used for anything. Parser is mysql/workbench/parser and that's fine.